mediafile.py

```diff
@@ -2326,6 +2327,20 @@ def bitrate(self):
                 return 0
             return int(self.filesize * 8 / self.length)
 
+    @property
+    def bitrate_mode(self):
+        """The mode of the bitrate used in the audio coding (a string).
+        Only available for certain file formats (empty where unavailable).
```

Comment:
We could "spoil the surprise" and admit that the only format it currently works for is MP3.

Reply:
Sure, I have updated the docstring